app/src/gui/components/notebook/index.tsx

```diff
@@ -233,100 +276,120 @@ export default function NotebookComponent(props: NotebookComponentProps) {
-                            project_id={project.project_id}
-                            metadata_key={'lead_institution'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                      <TableRow>
-                        <TableCell>
-                          <Typography variant={'overline'}>
-                            Project Lead
-                          </Typography>
-                        </TableCell>
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'project_lead'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                      <TableRow>
-                        <TableCell>
-                          <Typography variant={'overline'}>
-                            Last Updated
-                          </Typography>
-                        </TableCell>
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'last_updated'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                    </TableBody>
-                  </Table>
-                </TableContainer>
-              </Grid>
-              <Grid item xs={12} sm={12} md={12} lg={4}>
-                <RangeHeader
-                  project={project}
-                  handleAIEdit={handleNotebookTabChange}
+                  Edit
+                </Typography>
+              </IconButton>
+            </Box>
+
+            <Box sx={{p: 2}}>
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px'}}
+              >
+                <strong>Name:</strong>{' '}
+                <MetadataRenderer
+                  project_id={project.project_id}
+                  metadata_key={'name'}
+                  chips={false}
                 />
-              </Grid>
-            </Grid>
+              </Typography>
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px'}}
+              >
+                <strong>Description:</strong>{' '}
+                <MetadataRenderer
+                  project_id={project.project_id}
+                  metadata_key={'pre_description'}
+                  chips={false}
+                />
+              </Typography>
+
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px'}}
+              >
+                <strong>Status:</strong>{' '}
```

Comment:
This field doesn't have any useful meaning in the current system - it was there as a placeholder for some kind of published/draft status but that was never realised.   Perhaps hide this for now until we have a useful status flag.

Reply:
Sure thing.